Makefile

```diff
@@ -15,11 +15,11 @@ KUBECTL_VERSION = 1.10.1
 KUBECTL_BIN_SHA256SUM.linux = 1bb4d3793fb0f9e1cfee86599e0f43ae5f15578a01b61011fe7c9488e114a00b
 KUBECTL_BIN_SHA256SUM.darwin = 9484fd8a0cba513ab91fc192f6b6659d700f411502d01a7e02acb9f3b986037c
 
-HELM_VERSION = 2.8.2
-HELM_SRC_SHA256SUM.linux = 614b5ac79de4336b37c9b26d528c6f2b94ee6ccacb94b0f4b8d9583a8dd122d3
-HELM_SRC_SHA256SUM.darwin = a0a8cf462080b2bc391f38b7cf617618b189cdef9f071c06fa0068c2418cc413
-HELM_BIN_SHA256SUM.linux = 0521956fa22be33189cc825bb27b3f4178c5ce9a448368b5c81508d446472715
-HELM_BIN_SHA256SUM.darwin = 49c0f595f1c02c466191aa41143d623b553cc0d1fc2c92ab41984699aa3d0317
+HELM_VERSION = 2.9.1
```

Comment:
I'm glad we're going with 2.9.1. I had issues in 2.9.0 because of "Fix tiller deployment on RBAC clusters" that was reverted in the 2.9.1 update.

LGTM